view/frontend/web/js/view/payment/method-renderer/adyen-paypal-method.js

```diff
@@ -76,6 +76,10 @@ define(
                         street: formattedShippingAddress.street
                     };
                 }
+
+                // Paypal requires extra configuration
```

Comment:
Is this related to the GooglePay?

Reply:
This was a fix required for PayPal that I also included in this PR